packages/rspack-cli/src/utils/loadConfig.ts

```diff
@@ -4,7 +4,7 @@ import fs from "fs";
 import { RspackCLIOptions } from "../types";
 import { RspackOptions, MultiRspackOptions } from "@rspack/core";
 
-const supportedExtensions = [".js", ".ts"];
```

Comment:
what about cjs?

Reply:
unless people ask

Reply:
seems webpack supports cjs and i will add it